src/taoensso/sente.cljx

```diff
@@ -81,6 +81,16 @@
   (:require-macros
    [cljs.core.async.macros :as asyncm :refer (go go-loop)]))
 
+#+cljs
+(def is-node? (nil? enc/js-?win))
```

Comment:
Might `(= *target* "nodejs")` be appropriate for this? Zero experience with node, so not sure.